### PR TITLE
added constant in PngModule

### PR DESCRIPTION
**Pull request by marti1125**
Hi @carlwilson 
I am working in magic number of
extramodules/it/eng/jhove/module/png/PngModule.java

but I know what is the correct name  for: 
>  '4' is a magic number.
> case 4:

### CODE REVIEW

**Comment by carlwilson** [comment]
Hi @marti1125, thanks for this. The case statement switches on the value of "colour type", a single byte value, described in the specification here, section 4.1.1: https://www.w3.org/TR/PNG-Chunks.html.

**Comment by marti1125** [comment]
@carlwilson I added constants for color type, could you check if is it has correct name? please

**Review by carlwilson — APPROVED** [review]
Perfect, thanks

File: extramodules/it/eng/jhove/module/png/PngModule.java [hunk 1]
```diff
@@ -155,6 +155,15 @@
     // Standard keyword for the creation timestamp
     private final static String CREATION_TIME_KEYWORD = "Creation Time";
 
+    private final static int MAX_INT = 0x7FFFFFFF;
+
+    // Color Type
+	private final static int EachPixel_GRAYSCALE = 0;
+	private final static int EachPixel_RGB = 2;
+	private final static int EachPixel_PALETTA = 3;
```
Comment (carlwilson) [inline extramodules/it/eng/jhove/module/png/PngModule.java]: Small typo perhaps? EachPixel_PALETTE might be better, sorry.

**Review by carlwilson — CHANGES_REQUESTED** [review]
Sorry, I should have waited until I finished my tea before checking.

File: extramodules/it/eng/jhove/module/png/PngModule.java (hunk 1 above)
Comment (marti1125) [reply extramodules/it/eng/jhove/module/png/PngModule.java]: yep :) the problem when I was programming at midnight

**Comment by marti1125** [comment]
@carlwilson could you check again please :)

**Review by carlwilson — APPROVED** [review]
Good work, merging shortly.

**Pull request merged**